gobcore/logging/logger.py

```diff
@@ -16,6 +16,11 @@
 
 from gobcore.utils import gettotalsizeof
 
+# Custom log levels for data logging
+DATAINFO = 21
```

Comment:
Dit zijn magic numbers. Ik zou hier afleiden van logging constanten, bv logging.INFO + 1, of misschien een custom range?

Reply:
Done